Update serverless-sql-databases-overview.mdx (#3379)
scaleway/docs-content: serverless/sql-databases/reference-content/serverless-sql-databases-overview.mdx

## Technical Limitations

- SQL request size is limited to 512 KB. A single SQL query sent to a Serverless SQL Database cannot exceed 512 KB, but the response to the query is not limited and results tables can exceed 512 KB.

> - Total prepared statement size are limited to 512 KB per client connection.

Reviewer: ```suggestion
- Prepared statement size is limited to 512 KB per client connection.
```
